squirrel.worker/src/main/java/org/dice_research/squirrel/worker/impl/WorkerImpl.java

```diff
@@ -282,11 +286,24 @@ public void performCrawling(CrawleableUri uri) {
                 LOGGER.info("Crawling {} is not allowed by the RobotsManager.", uri);
                 activity.addStep(manager.getClass(), "Decided to reject this URI.");
             }
+
+
+        // Adding true label value for the prediction in uri map
+        if (activity.getNumberOfTriples()>0) {
+            uri.addData(Constants.URI_TRUE_LABEL,1);
+        } else {
+            uri.addData(Constants.URI_TRUE_LABEL, 0);
+        }
```

Comment:
Here, you are mixing up the binomial classification with the multinomial classification, right?

Reply:
URI_TRUE_LABEL is required if the user wants to perform binary classification and URI_TRUE_CLASS for multi class classification. Both these values have been modified to store string values containing the class name of the URI